api-client/src/endpoints/__tests__/createDisclaimerEndpoint.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('disclaimer', () => {
         path: '/augsburg/en/disclaimer/feedback-contact-and-opportunities-to-take-part',
         title: pageJson.title,
         content: '<div>Some disclaimer test content :)</div>',
-        lastUpdate: moment.tz('2017-06-12 12:27:57', 'GMT'),
+        lastUpdate: DateTime.fromJSDate(new Date('2017-06-12 12:27:57')),
```

Comment:
when I change it to `toISO`, it is returning slightly different `ts` on testing`"format": Object {},
          "standalone": Object {},
        },
      },
      "o": null,
-     "ts": 1689001409079,
+     "ts": 1689001409068,
      "weekData": null,
    },`